### PR TITLE
TO_CHAR conversion function.

### PR DESCRIPTION
**Pull request by Serdaro**
Related to #703

https://hazelcast.atlassian.net/jira/software/projects/DOCS/boards/60/timeline?selectedIssue=DOCS-442&text=to_char

### CODE REVIEW

**Review by rebekah-lawrence — APPROVED** [review]
LGTM

File: docs/modules/sql/pages/functions-and-operators.adoc [hunk 1]
```diff
@@ -357,6 +358,352 @@ Conversion functions allow you to convert the result type of one expression to a
...
+In the normal and exponential forms, if there is no negative sign provision and there is at
+least one digit position, an `M` pattern is prepended to the integer part. Similarly,
+if only one part has the `BR` and/or `B` patterns, the latest bracket in the order
+of processing is inserted to the opposite part. The inferred sign is inserted so that it
+encloses all non-fixed patterns in the part to which it is inserted.
+ 
+Lowercase variants of patterns are also accepted. If there is no special meaning of
+the lowercase variant, it has the same effect as its uppercase version.
+
+The `FM` pattern enables the fill mode, which suppresses padding.
```
Comment (burakgok) [inline docs/modules/sql/pages/functions-and-operators.adoc:677]:
```suggestion
The `FM` pattern enables _the fill mode_, which suppresses padding.
```

**Review by burakgok — CHANGES_REQUESTED** [review]

**Review by burakgok — APPROVED** [review]

**Comment by rebekah-lawrence** [comment]
@burakgok - out of curiosity, why the italics?

**Comment by burakgok** [comment]
@rebekah-lawrence The term "fill mode" was coined by Oracle and is very counter-intuitive. When you enable it, it suppresses padding, so the output takes the minimum amount of space possible, which is the exact opposite of what I expect from "the fill mode". The reason we use it in our documentation is to facilitate the migration from Oracle to Hazelcast SQL.

**Comment by rebekah-lawrence** [comment]
Thanks @burakgok - a new bit of knowledge to add to my brain. Much appreciated.

**Pull request merged**